docs/guides/developer/docs/installation/source-linux.md

```diff
@@ -25,6 +25,7 @@ Required:
     ffmpeg >= 3.2.4
     maven >= 3.1
     python >= 2.6, < 3.0
+    firefox/chrome/some other major browser
```

Comment:
So we don't know exactly which browsers work and which don't, right?

Reply:
It tries to detect all of them, but I'm hedging in the docs to keep maintenance low :)